db/src/main/resources/sql/schema/claims_list.sql

```diff
@@ -11,13 +11,11 @@
 -- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
-
```

Comment:
It doesn't like blank lines between different statements, huh?  Is that an option?  I think having blank lines in schema files makes things more readable.

Reply:
I agree that it'd make it more reasonable but there aren't really configurable options in this formatter.

Other formatters that I've seen that have configurable options still have issues like the extra spaces at the beginning of each line (this was a common theme among a few formatters, strangely enough)